proto/info/edge_node_cluster.proto

```diff
@@ -269,3 +272,37 @@ enum KubeComp {
   KUBE_COMP_CDI = 5;
   KUBE_COMP_LONGHORN = 6;
 }
+
+// KubeVMIStatus is the VirtualMachineInstancePhase
+// Refer to: https://pkg.go.dev/kubevirt.io/api/core/v1#VirtualMachineInstancePhase
+enum KubeVMIStatus {
+  KUBE_VMI_STATUS_UNSPECIFIED = 0;
+  KUBE_VMI_STATUS_PENDING = 1;
+  KUBE_VMI_STATUS_SCHEDULING = 2;
+  KUBE_VMI_STATUS_SCHEDULED = 3;
+  KUBE_VMI_STATUS_RUNNING = 4;
+  KUBE_VMI_STATUS_SUCCEEDED = 5;
+  KUBE_VMI_STATUS_FAILED = 6;
+  KUBE_VMI_STATUS_UNKNOWN = 7;
+}
+
+// KubeVMIInfo is the VMI status of eve VM apps
+message KubeVMIInfo {
+  // Name of the VirtualMachineInstance
+  string name = 1;
+
+  // Status (VirtualMachineInstance.Status.Phase)
+  KubeVMIStatus status = 2;
+
+  // Creation Time of the VirtualMachineInstance
+  google.protobuf.Timestamp creation_time = 3;
+
+  // LastTransitionTime is the last time the Ready Condition was true
```

Comment:
Does "the Ready Condition" refer to the is_ready field below? If so it makes sense to update this comment.
And I'd say "was set to true" and not "was true".  Would the timestamp also be updated when it is set to false?

Reply:
I just fixed the comments the current comment does leave a hole.  The timestamp should be updated for both ->ready and ->not-ready transitions.